dayrepo/snippets/templates/snippet_list.html

```diff
@@ -14,6 +15,8 @@ <h2>提出済み日報一覧</h2>
     <button onclick="location.href='{% url 'excelfile_download' post.id %}'">出力</button>
     <button onclick="location.href='{% url 'checklist_edit' post.checklist_id.id %}'">点検編集</button>
     <button onclick="location.href='{% url 'snippet_edit' post.id %}'">日報編集</button>
+    <button onclick="location.href='{% url 'post_delete' post.id 'snippet' %}'">日報削除</button>
+    <button onclick="location.href='{% url 'post_delete' post.id 'all' %}'">オール削除</button>
```

Comment:
一旦画面上でだけの命名になりますが、以下でどうですか？
```suggestion
    <button onclick="location.href='{% url 'post_delete' post.id 'all' %}'">一括削除</button>
```

Reply:
ありっすね